wiki/markdown/usage/service.md

```diff
@@ -139,7 +139,133 @@ Status codes are generated according to the below table.
 | 503         | Service Unavailable   | "No protocol service has been found registered"                                                                               | When specified message protocol is not loaded                                                                                                                                   |
 | 207         | Multi Status          | Multi-Status                                                                                                                  | Is returned when there's a partial success, i.e. some events failed, some successfully generated/published, return status should be 207.                                        |
 
+### Examples:
```

Comment:
Please, provide responses, so user can see how the outputs are structured.

Reply:
Done